recipes/rabbit.php

```diff
@@ -26,12 +26,13 @@
         $config['message'] = "Deployment to '{$host}' on *{$prod}* was successful\n($releasePath)";
     }
 
-    $defaultConfig = array(
+    $defaultConfig = [
```

Comment:
Could you use PHP 5.3 syntax? We want to have recipes compatible with deployer 2.x